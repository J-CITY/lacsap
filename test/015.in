
program HelloWorld;
var
    _int, lazy_town_ : integer;
    __r : real;
begin

  if 'abc' = 'adadasd then
    _int = 1;
end.
