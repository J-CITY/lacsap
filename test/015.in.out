ERROR: ERROR FORGOT TILDA (-1008) row: 0 col: 0
